bedrock/firefox/templates/firefox/pocket.html

```diff
@@ -25,39 +25,31 @@
 
 {% block content %}
 <main class="mzp-t-firefox">
-  <section id="pocket-hero" class="mzp-c-hero mzp-has-image mzp-t-dark">
-    <div class="mzp-l-content">
-      <div class="mzp-c-hero-body">
-        <h1 class="mzp-c-hero-title">
-          <img src="{{ static('img/logos/pocket/pocket-wordmark-white.svg') }}" alt="{{ _('Pocket') }}" width="125" height="33">
-          {{ _('An Award-Winning App From Firefox') }}
-        </h1>
-        <div class="mzp-c-hero-desc">
-          <p>{{ _('Over 10 million users rely on Pocket to discover and savor the best articles, news, stories and videos. And as a member of the Firefox family, privacy is paramount.') }}</p>
-        </div>
-
-        <div class="mzp-c-hero-cta">
-          <p>{{ pocket_fxa_button(entrypoint=_entrypoint, button_text=_('Try Pocket Now'), optional_parameters={'s': _source}, optional_attributes={'data-cta-text': 'Try Pocket Now', 'data-cta-type': 'activate pocket', 'data-cta-position': 'primary'}) }}</p>
-        </div>
-      </div>
-    </div>
-
-    <div class="mzp-c-hero-image">
-      {{ high_res_img('img/firefox/pocket/pocket-hero.png', {'alt': ''}) }}
-    </div>
-  </section>
-
-  <div id="pocket-popularity" class="mzp-l-content">
-    {% call feature_card(
-      title=_('Capture What Fascinates You'),
-      image_url='img/firefox/pocket/pocket-illo-reading.png',
-      include_highres_image=True,
-      aspect_ratio='mzp-has-aspect-16-9',
-      class='mzp-l-card-feature-right-third'
-    ) %}
-      <p>{{ _('Turn your downtime into quality time. Pocket is the place to save, read and get fueled by the best content on the web.') }}</p>
-    {% endcall %}
-  </div>
+  {% call split(
+    id='pocket-hero',
+    image_url='img/firefox/pocket/pocket-hero.png',
+    include_highres_image=True,
+    block_class='pocket-hero mzp-l-split-center-on-sm-md mzp-l-split-hide-media-on-sm-md mzp-l-split-pop-bottom',
+    media_class='mzp-l-split-media-overflow mzp-l-split-v-end',
+    theme_class='mzp-t-dark'
+  ) %}
+    <h1>
+      <div class="mzp-c-wordmark mzp-t-wordmark-sm mzp-t-product-pocket">Pocket</div>
+      {{ _('An Award-Winning App From Firefox') }}
+    </h1>
+    <p>{{ _('Over 10 million users rely on Pocket to discover and savor the best articles, news, stories and videos. And as a member of the Firefox family, privacy is paramount.') }}</p>
```

Comment:
Is this something that needs to be localized? I just haven't seen that many instances of plain text

Reply:
This page isn't currently localized, and I don't think it ever will be. At least not this version; we could update it at some point and a new one might be localized.

The `{{ _(' '} }}` notation is how the old l10n system encapsulated strings (also in `{% trans %}{% endtrans %}` blocks), and we used to mark up pretty much all strings that way just in case we decided to translate something later. So you'll see them in older templates when it's an unlocalized page that wasn't converted to Fluent.